examples/layers/preprocessing/bounding_box/jittered_resize_demo.py

```diff
@@ -23,6 +23,7 @@ def main():
         target_size=(512, 512),
         scale_factor=(3 / 4, 4 / 3),
         bounding_box_format="xyxy",
+        minimum_box_area_ratio=0.5
```

Comment:
remove this as well. Maybe add a doc string to `clip_to_image`, explain the args and add an example there